### ISSUE TITLE
Broken docs links

### ISSUE THREAD

**Issue by azim-b** [h2oai/wave#1212]
Docs links for `Components` seem to be broken. I suspect this is due to the introduction of `Widgets` in the docs.

For example:
https://wave.h2o.ai/docs/components/#dropdown

![image](https://user-images.githubusercontent.com/64787868/151903778-33b2bfe2-3eb2-414c-80a4-7c9cd984d3b0.png)

is now: https://wave.h2o.ai/docs/widgets/form/dropdown

**Comment by mturoci** [comment]
Did we ever support `/components` link? I don't remember. But that's right, such links ade not supported anymore.

@azim-b did you find the broken link above somewhere in the docs?

**Comment by azim-b** [comment]
yes, I believe we did. I have used the `Components` guide many times.

And yes, if you search for any component, it still links to the old URL:

<img width="469" alt="image" src="https://user-images.githubusercontent.com/64787868/151933292-581a9f7e-0eaf-4735-be13-45120ec39b74.png">

there's also a search result for `Components` in `Guide`, which is broken too (essentially all component links may be broken).
